app/assets/stylesheets/components/_card-show.scss

```diff
@@ -1,10 +1,11 @@
 .wrapper-one-lightblue {
-  display:flex;
-  align-items:center;
+  display: flex;
+  align-items: center;
```

Comment:
I've got rid of this line on my version, because I've added a header to the map section, and as the map won't always have the same size as the "long description" div, it's preferable to get everything aligned on top. If we don't use a header, then we can indeed use it to get the map centered.